test/testlib/minikube_utilities.go

```diff
@@ -756,6 +788,35 @@ func GetAdminEventLog(t *testing.T, namespace string, podName string) {
 	)
 }
 
+func GetFile(t *testing.T, namespace string, podname string, path string, filename string) error {
+	pwd, err := os.Getwd()
+	require.NoError(t, err)
+
+	fromPath := fmt.Sprintf("%s/%s:%s/%s", namespace, podname, path, filename)
+	dirPath := filepath.Join(pwd, RESULT_DIR, namespace, podname)
+	toPath := filepath.Join(dirPath, filename)
+
+	_ = os.MkdirAll(dirPath, 0700)
+
+	f, err := os.Create(toPath)
```

Comment:
We have to explicitly create the file before running `kubectl cp` with that file as the destination? Or is this just to truncate the file? Either way, I'm not sure why we would defer the call to `f.Close()` rather than executing eagerly.